io.kinvolk.Headlamp.yaml

```diff
@@ -0,0 +1,85 @@
+app-id: io.kinvolk.Headlamp
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.golang
+- org.freedesktop.Sdk.Extension.node12
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+command: run-headlamp.sh
+separate-locales: false
+rename-desktop-file: headlamp.desktop
+rename-icon: headlamp
+finish-args:
+- --share=ipc
+- --socket=x11
+- --socket=wayland
+- --device=dri
+- --share=network
+# We use .kube/config to know what clusters/contexts are set up
+- --filesystem=~/.kube/config:ro
+# It's common to run minikube for experimenting, so it should be fine to read this dir
+- --filesystem=~/.minikube:ro
+# Config folder
+- --filesystem=~/.config/Headlamp:ro
+modules:
+- name: headlamp
+  buildsystem: simple
+  build-options:
+    append-path: ":/usr/lib/sdk/node12/bin:/usr/lib/sdk/golang/bin:"
+    env:
+      XDG_CACHE_HOME: /run/build/headlamp/flatpak-node/cache
+      npm_config_cache: /run/build/headlamp/flatpak-node/npm-cache
+      npm_config_nodedir: /usr/lib/sdk/node12
+      npm_config_offline: 'true'
+      npm_config_no_save: 'true'
+      npm_config_loglevel: notice
+      GOPATH: /run/build/headlamp/
+      GO111MODULE: "auto"
+      GOFLAGS: "-mod=vendor"
+      DEBUG: electron-builder
+  sources:
+  - type: git
+    url: https://github.com/kinvolk/headlamp.git
+    branch: master
```

Comment:
Could you pin a specific commit here? Just so the build is reproducible. Other than that, LGTM.

Reply:
Makes sense. Will do. Thanks.